assets/scss/tools/_string.scss

```diff
@@ -8,3 +8,66 @@
 
     @return $string;
 }
+
+// https://stackoverflow.com/questions/32376461/how-to-split-a-string-into-two-lists-of-numbers-in-sass
+@function str-split($string, $separator) {
```

Comment:
Can we add some doc blocks for these helpers which we are adding. Also this would be a good candidate for documentation.